### PR TITLE
Updates mobility and contact code

### PR DESCRIPTION
**Pull request by cash**
* Adds mobility matrix as weighted edges. Assumes the mobility matrix is located at Outputs/XXX/MobilityMatrix/XXX_mobility_matrix.csv (change that to whatever you want)
* Sets the diagonal of the mobility matrix to 1
* Updated the population and contact matrix to 16 divisions.
* Adds population total to graph metadata

Let me know if you have any questions on these.

We also have code for filling in missing values for the COVID case data that we'll get to you soon.

### CODE REVIEW

File: Generate_graph.py [hunk 1]
```diff
@@ -69,14 +71,33 @@ def main(country_iso3):
...
 
+def initialize_with_mobility(main_dir, country_iso3):
+    filename = os.path.join(main_dir, MOBILITY_DIR, MOBILITY_FILENAME.format(country_iso3=country_iso3))
+    logger.info(f'Reading in mobility from {filename}')
+    mobility = pd.read_csv(filename)
+    mobility.set_index("ADM", inplace=True)
+
+    # set diagonal to 1 (should this be moved into code that produces mobility matrix?)
+    for adm in mobility.index:
+        mobility[adm].loc[adm] = 1
```
Comment (turnerm) [inline Generate_graph.py:82]: Is the diagonal not set to 1 already? This should be taken care of in the mobility matrix repo

Comment (cash) [reply Generate_graph.py:82]: It wasn't when I grabbed the data. Do you want me to make a new PR without this commit?

Comment (turnerm) [reply Generate_graph.py:82]: Ah sorry, so the latest matrix is [here](https://github.com/OCHA-DAP/pa-movement-patterns-matrix/blob/master/output/afg/afg_mobility_matrix.csv). 

No need to make a new PR I think, just make the change on this branch

File: Generate_graph.py [hunk 2]
```diff
@@ -86,13 +107,15 @@ def add_exposure(G, main_dir, country_iso3):
...
         'ADM2_PCODE',
+        'tot_sad',
         'group_pop_f',
         'group_pop_m'
     ]
     exposure = exposure[columns]
     # Rename some
     rename_dict = {
         'ADM2_EN': 'name',
+        'tot_sad': 'population',
```
Comment (turnerm) [inline Generate_graph.py:118]: I will make a PR to add population and pop density computed from summing the age / gender data, so you can leave this out (as well as line 110)

**Comment by cash** [comment]
I'm closing this and making separate PRs for the commits that are still relevant given comments by @turnerm.

**Pull request closed without merging**